### DISCUSSION TITLE
Make WALLS count as a barrier for the perceptive

### DISCUSSION THREAD

**Discussion by spitfire1081** [Saibot393/perceptive#79] [Ideas]
Sometimes we roll stealth for the NPCs in a dungeon, and the players are able to revel them behind walls. It would be nice to have an option to make tokens undetectable behind full cover, inside a room with closed doors, for example

**Comment by Saibot393** [comment]
Wall should already block vision for this purpose, i will look into and see why this happens.

**Comment by Saibot393** [comment]
Assuming you are using the levels the bug should be fixed. There was a small error in my levels LOS calculations
